src/components/ui/Link/link.scss

```diff
@@ -0,0 +1,40 @@
+@import "../../../styles/vendors/include-media.scss";
+
+[data-store-link] {
+  min-width: auto;
+  min-height: auto;
+  padding: var(--space-2) var(--space-0);
+  text-decoration: none;
+  &:hover { text-decoration: underline; }
+  &:visited { color: var(--color-link-visited); }
+}
+
+// Links ////////////////////////////
+
+[data-link-inverse] {
+  color: var(--color-link-inverse);
+  &:visited { color: var(--color-link-inverse); }
+}
+
+.link {
+  color: var(--color-link);
+  line-height: 1.5;
+}
+
+.link-display {
```

Comment:
Hmm.. if we are going to follow BEM syntax, are we considering these following classes elements or modifies? 
`link__display` or `link--display`?

Reply:
Or maybe we want to pass all this classes as variants?